src/connector.js

```diff
@@ -356,46 +368,40 @@ function initTpl() {
 	if ( !suggestionsElement && searchBoxElement && params.unsupportedSuggestions && params.numberOfSuggestions > 0 ) {
```

Comment:
`unsupportedSuggestions` can be removed
```suggestion
	if ( !suggestionsElement && searchBoxElement && params.numberOfSuggestions > 0 ) {
```